coldfront/plugins/qumulo/management/commands/migrate_allocation_attributes.py

```diff
@@ -32,9 +32,8 @@ def _migrate_allocation_attribute(self, attribute_name, default_value):
         )
 
         for allocation in all_storage_2_allocations:
-            if getattr(allocation, attribute_name, None) is None:
-                AllocationAttribute.objects.create(
-                    allocation_attribute_type=attribute_type,
-                    allocation=allocation,
-                    value=default_value,
-                )
```

Comment:
@peaches7083 did you mean to do the following?

`                AllocationAttribute.objects.create(
                    allocation_attribute_type=attribute_type,
                    allocation=allocation,
                    defaults: {value=default_value},
                )`

Reply:
No (?).  What would be the reason to do it that way?